build/scripts/build-client-binary

```diff
@@ -33,8 +33,8 @@ gobuild_component() {
         build_command="cd /tmp/go/src/${extract_path} && make build"
     elif [ ${component} = 'node_exporter' ] || [ ${component} = 'mysqld_exporter' ] || \
          [ ${component} = 'proxysql_exporter' ] || [ ${component} = 'rds_exporter' ] || \
-         [ ${component} = 'postgres_exporter' ]; then
-        build_command="cd /tmp/go/src/${extract_path} && go build -o \${binary_dir}/${result_file} ./${component_path}"
+         [ ${component} = 'postgres_exporter' ] || [ ${component} = 'azure_metrics_exporter' ]; then
+        build_command="cd /tmp/go/src/${extract_path} && make feature-build"
```

Comment:
let's make it just `make build` or `make release`

Reply:
`make build` already taken in some exporters so i will use `make release`

Reply:
done